guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,22 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select Schedule Remote Job.
+. In the *Job category* field, select `Convert to RHEL`.
+. In the *Job template* field, verify that the Job Template is set to `Convert to RHEL`.
+. On the *Job Invocation* page, define the main job settings.
+. The *Search Query* field displays the name of any search you have preformed, or the list of hosts you have selected.
+. In the *Activation Key* field, enter the host activation key.
+. Optional: Click *Show Advanced Fields* if you want to configure further aspects.
+. Click *Restart*.
+. Note that:
+* The conversion can take some time.
+* During the conversion, the host reboots.
+* After the reboot, new OS facts are sent to {Project}.
+* When complete, the OS for the host changes to RHEL 7 and the subscription status turns green.
```

Comment:
Probably want to use `{RHEL}`. Does this script only work for EL7> RHEL7, or also EL8 > RHEL8? Maybe drop the `7`? And maybe add this piece of information to the introduction.

Reply:
Not sure. Checking.